lynx/common/scene.py

```diff
@@ -11,13 +11,27 @@
 class Scene(Serializable):
     entities: List[Entity] = field(default_factory=list)
     _objects_map: Dict[Vector, Object] = field(default_factory=dict)
+    _id_map: Dict[int, Object] = field(default_factory=dict)
```

Comment:
Please rename to `_position_entities_map` and `_id_entities_map`.

Reply:
These are quite long, maybe `_position_map` and `_id_map`?

Reply:
I wouldn't want those, because they sound as if we only kept positions and ids in these dicts. And that's not true, because entities are the focus.
Long names are not a problem at all, as long as their descriptive.

Reply:
Not a huge fan of long names, but I guess u're right :). What do you think bout _object_position_map + _object_id_map? I think I rather having it in a format `value_key_map`